### PR TITLE
feat: add sample code for expense parser (folder expense-parser-python)

### PR DESCRIPTION
**Pull request by jiya-zhang**

### CODE REVIEW

File: community/expense-parser-python/cloud-functions/main.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
+import base64
+import re
```
Comment (galz10) [inline community/expense-parser-python/cloud-functions/main.py:2]: I don't think you use this

File: community/expense-parser-python/cloud-functions/main.py [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+import os
+import json
+from datetime import datetime
```
Comment (galz10) [inline community/expense-parser-python/cloud-functions/main.py:5]: I don't think you use this

File: community/expense-parser-python/table-schema/bq_schema.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      "mode": "NULLABLE",
+      "name": "type_",
+      "type": "STRING"
+    },
+    {
+      "mode": "NULLABLE",
+      "name": "mention_text",
+      "type": "STRING"
+    }
+  ]
```
Comment (galz10) [inline community/expense-parser-python/table-schema/bq_schema.json:22]: add new line

**Review by galz10 — CHANGES_REQUESTED** [review]
After fixing the few comments could you run pylint and fix all the lint errors

File: community/expense-parser-python/README.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+![GCP Workflow](https://user-images.githubusercontent.com/47513414/150703075-8f608859-436e-4c22-8dc9-f71121705f3a.png)
+
+# GCP Services used in the Demo
+* [Google Cloud Procurement Document AI](https://cloud.google.com/solutions/procurement-doc-ai)
+* [Google Cloud Storage](https://cloud.google.com/storage)
+* [Google Cloud Functions](https://cloud.google.com/functions)
+* [BigQuery](https://cloud.google.com/bigquery)
+
+# Steps to re-create this demo in your own GCP environment
+1. Create a Google Cloud Platform Project
```
[the remark this answers is not in the document]
Comment (jiya-zhang) [reply community/expense-parser-python/README.md:16]: Can I just specify that they should use Cloud Shell?

**Comment by kweinmeister** [comment]
Hi @jiya-zhang - good news, we've automatically resolved several of the linting issues with the OwlBot post-processor.

Please also ensure that your files include a copyright header at the top.. You can look at other examples in the repo.

Thanks!

File: community/expense-parser-python/README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (galz10) [reply community/expense-parser-python/README.md:16]: I think using the cloud cli is better then Cloud shell for this but yes you can

**Review by kweinmeister — APPROVED** [review]

**Pull request merged**